lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -124,6 +127,36 @@ export class BrowserCacheManager extends CacheManager {
         }
     }
 
+    /**
+     * Searches all cache entries for MSAL accounts and creates the account key map
+     * This is used to migrate users from older versions of MSAL which did not create the map.
+     * @returns 
+     */
+    private createAccountKeyMap(): void {
+        const accountKeys = this.getItem(StaticCacheKeys.ACCOUNT_KEYS);
```

Comment:
This defaults to whatever memory option is set, which implies we need to serialize and deserialize the accountKeys every time we load the page. Same q as above, concerned if we ever cross size limits, other wise looks good.